cmd/check_container.go

```diff
@@ -226,6 +228,24 @@ var checkContainerCmd = &cobra.Command{
 	},
 }
 
+func pyxisHostLookup(pyxisEnv, hostOverride string) string {
+	envs := map[string]string{
+		"prod":  "catalog.redhat.com/api/containers",
+		"uat":   "catalog.uat.redhat.com/api/containers",
+		"qa":    "catalog.qa.redhat.com/api/containers",
+		"stage": "catalog.stage.redhat.com/api/containers",
+	}
+	if hostOverride != "" {
+		return hostOverride
+	}
+
+	pyxisHost, ok := envs[pyxisEnv]
+	if !ok {
+		pyxisHost = envs["prod"]
+	}
+	return pyxisHost
```

Comment:
/lgtm

Sorry, I totally didn't think about this before. I know the default behavior is to use prod, but if the user is overriding this, then in theory they don't want prod. Should we default to a non-prod environment of some kind in cases where the user specified this flag, but with an invalid value? 

Don't know if I like it (vs. throwing an error), but anyway. Marking this LGTM otherwise.

Reply:
In practice, I would agree with you @komish ie throw the error. For this use-case where there is `internal` and `external` hosts, I like having this override so as a dev team we can see if the issues is with Pyxis itself or just the `external` instance we call.